framework/addons/data/addons.json

```diff
@@ -679,10 +679,40 @@
                 "status": "none",
                 "text": "Metadata information will not be registered."
             }
+        },
+        "RushFiles": {
```

Comment:
https://github.com/tsukaeru/RDM-osf.io/blob/feature/rushfiles/framework/addons/data/addons.json#L683
https://github.com/tsukaeru/RDM-osf.io/blob/feature/rushfiles/addons/rushfiles/apps.py#L12
https://github.com/tsukaeru/RDM-osf.io/blob/2197b054c0ec41fac26f89b69019b3c670ccb53f/addons/rushfiles/static/node-cfg.js#L6

Those symbols must match. (`FileBako` or `RushFiles`)

Reply:
Changed here to `FileBako`.